### DISCUSSION TITLE
Skip a workflow to restart a specific task

### DISCUSSION THREAD

**Discussion by Aerglonus** [Zibbp/ganymede#405] [Q&A]
So recently I got a live archive where for some reason the `Video convert` task didn't trigger  making the workflows to fail, re-running them will fail at `DownloadTwitchLiveThumbnailsWorkflow` since the VOD for that stream is not public but the thumbnail is already on the directory so there's no need  in this case to redownload the thumbnail and I did tried through the option on "Tasks > Queue Tasks > Start stuck queue items" but this wasn't triggering anything so the only way I was able to rerun the "Video convert" task was using the API  `/api/v1/archive/restart`.

In cases like this where the temporal file exists but the  convert video task didn't trigger, Is there a way to skip a given workflow task or manually restart a task (e.g. Video Convert)  directly from the UI ?

**Comment by Zibbp** [comment]
So the video convert task did not start *after* the video download task?
Currently nothing exists in the frontend to start a task that never started. I'll look into getting something setup for this.

**Reply by Aerglonus** [reply]
> So the video convert task did not start after the video download task?

Yeah, no error at all it was just chillin there doing nothing because the workflows stopped as if the "Download thumbnail" task failed but it didn't.
